### DISCUSSION TITLE
Setting up a Brazil profile not working

### DISCUSSION THREAD

**Discussion by FEndo** [audiamus/BookLibConnect#145] [General]
I've been trying to set up a profile for my Brazilian account, but every time I log in, I end up on the https://www.amazon.com.br/ap/404 page.

I also have an old US account where I managed to register a new device and access my library using BookLibConnect. However, with my Brazilian account, I haven't been able to get the URL for registering a new device with BookLibConnect.

I don't think the log will show anything since this seems to be an issue outside of this app. Any advice or suggestions would be greatly appreciated!

**Comment by audiamus** [comment]
Yes, it's different for Brazil. Your have to intercept the http response. I cant's provide you with precise instructions, but other users have managed, see here: #133

**Reply by FEndo** [reply]
Perfect! It worked, thanks a lot!
